contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -393,8 +393,8 @@ WHERE (pg_has_role(suser_id(), 'sysadmin'::TEXT, 'MEMBER')
 UNION ALL
 SELECT
 CAST('public' AS SYS.SYSNAME) AS name,
-CAST(-1 AS INT) AS principal_id,
-CAST(CAST(0 as INT) as sys.varbinary(85)) AS sid,
+CAST(2 AS INT) AS principal_id,
```

Comment:
Verify the 'public' role configuration across different SQL Server system views including sys.database_principals and user_token.